### ISSUE TITLE
Override of method 'initialize()' not permitted by Swift 4

### ISSUE THREAD

**Issue by mazz** [Swinject/SwinjectStoryboard#76]
This seems to be finally removed in Swift 4. Any suggested workarounds? This is a compile error in `Storyboard+Swizzling.swift`

```
... /Pods/SwinjectStoryboard/Sources/Storyboard+Swizzling.swift:25:30: error: method 'initialize()' defines Objective-C class method 'initialize', which is not permitted by Swift
    open override class func initialize() {
```

**Comment by yoichitgy** [comment]
Are you using SwinjectStoryboard `1.1.2`?

**Comment by FaithfulAudio** [comment]
For some reason cocoapods is pulling 1.0.0

**Comment by jakubvano** [comment]
Closing this, as it should be resolved in `1.1.2`

**Comment by mazz** [comment]
Yes it's working fine after fixing Podfile. Sorry about false alarm.
